notebooks/HASP/Setup/Setup.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>Thanks @srosagomez. Navigation to 3.1 doesn't seem to work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>